tests/requirements/base.txt

```diff
@@ -6,6 +6,8 @@ jinja2 >= 2.7
 numpy
 Pillow
 PyYAML
+# Remove if it ends up being possible to build libmemcached on the Windows Jenkins box.
```

Comment:
I mean: Even with the `sys.platform != 'win32'`, if the Window box's CACHES settings contains the PyLibMCCache backend, we'll still have issues. As such, we'll need the Windows test run to use a different settings file.